libphobos/configure.ac

```diff
@@ -113,6 +113,9 @@ AC_SUBST(phobos_compiler_shared_flag)
 lt_prog_compiler_pic_D="$phobos_compiler_shared_flag"
 pic_mode='default'
 
+# Determine what GCC version number to use in filesystem paths.
+GCC_BASE_VER
```

Comment:
Is this part of gcc-8?

Reply:
Hm. Should I completely revert https://github.com/D-Programming-GDC/GDC/pull/668?